### PR TITLE
Fix/add transition props upgrade react

### PR DESCRIPTION
**Pull request by shanejonas**

### CODE REVIEW

**Comment by shanejonas** [comment]
^^ codecov doesnt like that i added some line breaks here

File: src/Methods/Methods.test.tsx [hunk 1]
```diff
@@ -39,11 +39,68 @@ it("renders schema methods name", () => {
       },
     ],
   };
-  ReactDOM.render(<Methods schema={schema as any}/>, div);
+  ReactDOM.render(<Methods schema={schema as any} disableTransitionProps={true} />, div);
   expect(div.innerHTML.includes("get_pet")).toBe(true);
   ReactDOM.unmountComponentAtNode(div);
 });
 
+it("doesnt renders collapsed contents", () => {
```
Comment (zcstarr) [inline src/Methods/Methods.test.tsx]:
```suggestion
it("doesnt render collapsed contents", () => {
```

File: src/Methods/Methods.tsx [hunk 2]
```diff
@@ -31,8 +31,8 @@ const styles = (theme: Theme) => ({
     fontSize: theme.typography.pxToRem(15),
   },
   root: {
-    marginBottom: theme.spacing.unit,
-    marginTop: theme.spacing.unit * 3,
+    marginBottom: theme.spacing(2),
+    marginTop: theme.spacing(3),
```
Comment (zcstarr) [inline src/Methods/Methods.tsx]: should this require a multiplier as well?

File: src/Methods/Methods.tsx [hunk 3]
```diff
@@ -50,22 +50,28 @@ interface IProps extends WithStyles<typeof styles> {
...
   public render() {
-    const { schema, classes, uiSchema } = this.props;
+    const { schema, classes, uiSchema, disableTransitionProps } = this.props;
     if (!schema) {
       return null;
     }
     const methods: MethodObject[] = schema.methods;
     const methodsExist = methods && methods.length > 0;
     if (!schema || !schema.methods || !methodsExist) { return null; }
+    console.log("uiSchema", uiSchema); // tslint:disable-line
```
Comment (zcstarr) [inline src/Methods/Methods.tsx]: should this be yanked?

File: src/Methods/Methods.tsx (hunk 2 above)
Comment (zcstarr) [reply src/Methods/Methods.tsx]: discussed offline. Theme update shifts spacing to a function, so the multiplication no longer seems necessary for the newer rendering methods.

File: src/Methods/Methods.tsx (hunk 3 above)
Comment (zcstarr) [reply src/Methods/Methods.tsx]:
```suggestion
```

**Pull request merged**